### ISSUE TITLE
Cannot serve page locally / build on travis

### ISSUE THREAD

**Issue by jakob-r** [mlr-org/mlr-org.github.io#17]
Somehow Jekyll has changed but github still runs in compatibility mode?

Github still can serve this site but I can not render it locally nor can travis build this site.

**Comment by giuseppec** [comment]
Maybe this https://github.com/yihui/blogdown-jekyll helps ?

**Comment by jakob-r** [comment]
Hmm. Then we should probably switch to HUGO and do the build by travis and say goodbye to the jekyll github integration.

https://www.metachris.com/2017/04/continuous-deployment-hugo---travis-ci--github-pages/

**Comment by ja-thomas** [comment]
can you currently build the site locally? 
At least for my setup there seem to be a ton of dependency issues

**Comment by ja-thomas** [comment]
At least @giuseppec was able to push another post, so in general it should work, right?

**Comment by jakob-r** [comment]
Yeah. It became more complicated over the time. I don't know. The github process still runs but it is complicated to serve the site locally for testing.

**Comment by pat-s** [comment]
I've written a post but its a pain to get it building locally. All the Jekyll deps drive me crazy. 

I would really need to preview it before posting. Somehow the post is not rendered as a draft at https://mlr-org.github.io/visualize-spatial-cv. Does somebody know why?

**Comment by pat-s** [comment]
Ok, its served meanwhile. But we should really try to simplify the local rendering process. Maybe even think of switching to HUGO / blogdown to simplify things.
